src/bentoml/_internal/runner/runner.py

```diff
@@ -305,8 +306,8 @@ def init_local(self, quiet: bool = False) -> None:
     def init_client(
         self,
         handle_class: type[RunnerHandle] | None = None,
-        *args: P.args,
-        **kwargs: P.kwargs,
+        *args: t.Any,
```

Comment:
What is the motivation behind changing this from P -> t.Any?

Reply:
The `P` here is not a valid usage of generic type var, it is not derived or reused elsewhere in this context.

Reply:
You are correct here. It seems like we `P` doesn't have any context here. fwiw I think this type change is not really relevant to this PR?

Maybe we can create a small separate PR to address the type issue?

Reply:
Since there are many more similar issues in the codebase so I prefer to make the change progressive along with such feature improvements. But if you think separating is a more appropriate I can update the PR.

Reply:
I don't have a strong opinion about this. I think that usually, it is probably nicer to separate out type change, which will help when you do `git blame` to see the change more meaningful.

But again, I don't have a strong opinion on this; and since you already included the change here, I think it is ok for this case.